plugin.xml

```diff
@@ -25,18 +25,25 @@
   <js-module src="www/com.salesforce.util.push.js" name="util.push"></js-module>
 
   <engines>
-    <engine name="cordova" version="&gt;=3.5.0" />
-    <engine name="cordova" version="&lt;=3.8.0" />
+    <engine name="cordova" version="&gt;=4.0.0" />
```

Comment:
Most changes are in this file. Everything else is script generated updates from the source repo.